scripts/publish_prod.sh

```diff
@@ -24,7 +24,7 @@ fi
 
 yarn login
 
-./scripts/run_tests.sh
+# ./scripts/run_tests.sh
```

Comment:
I still can't figure out why this script is breaking.

Reply:
oops didn't mean to commit this